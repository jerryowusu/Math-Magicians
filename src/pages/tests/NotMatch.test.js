/* eslint-disable jsx-quotes */
import React from 'react';
import renderer from 'react-test-renderer';
// import { BrowserRouter as Router } from 'react-router-dom';
import NotMatch from '../NotMatch';

it('should display the home page content 1', () => {
  const tree = renderer.create(<NotMatch />).toJSON();
  expect(tree).toMatchSnapshot();
});
