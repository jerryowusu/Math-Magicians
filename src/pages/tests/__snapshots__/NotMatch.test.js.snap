// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the 404 page if there is no page 1 1`] = `
<div>
    <h3>No match for this page</h3>
</div>
`;

exports[`should display the home page content 1 1`] = `
<div>
  <h3>
    No match for this page
  </h3>
</div>
`;
